styles.css

```diff
@@ -0,0 +1,635 @@
+  border-bottom: 0.15rem solid var(--primary-color);
+  padding-bottom: 0.5rem;
+}
+
+.about-program__heading span {
+  border-bottom: 0.15rem solid var(--primary-color);
+  padding-bottom: 1.5vw;
+}
+
+.about-program__text {
+  font-size: 0.8rem;
+}
+
+.logo-about {
+  border: 0.1rem solid var(--tertiary-color);
+  margin-bottom: 1.5rem;
+  align-items: center;
+}
+
+.about-program__logo {
+  font-family: cocogoose, sans-serif;
+  color: var(--gray-black);
+}
+
+.about-program_date {
+  font-size: 0.7rem;
+  border-top: 0.1rem solid var(--gray-black);
+  border-bottom: 0.1rem solid var(--gray-black);
+  color: var(--gray-black);
+}
+
+.last-summit {
+  text-align: center;
+}
+
+.last-summit__heading {
+  font-size: 1rem;
+  margin-top: 2rem;
+}
+
+.last-summit__text {
+  font-size: 1rem;
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+  padding: 0 4rem;
+}
+
+.last-summit__images {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+}
+
+.last-summit__image-wrapper {
+  width: 30vw;
+  height: 30vw;
+  position: relative;
+  text-align: center;
+}
+
+.last-summit__image-wrapper img {
+  width: 100%;
+  height: 100%;
+}
+
+.last-summit__info {
+  padding-top: 7vw;
+  width: 100%;
+  height: 100%;
+  font-size: 2.5vw;
+  line-height: 4vw;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: var(--white);
+  background: linear-gradient(to right, rgba(255, 0, 0, 0.7), rgba(252, 0, 0, 0.7));
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 100vw;
+  }
+
+  .burger {
+    display: none;
+  }
+
+  .sec-nav__list {
+    justify-content: flex-end;
+    gap: 2em;
+    align-items: center;
+    background-color: var(--secondary-color);
+    visibility: visible;
+    position: static;
+  }
+
+  .sec-nav__item:last-child {
+    margin-right: 2rem;
+  }
+
+  .sec-nav__list li {
+    color: var(--white);
+  }
+
+  .sec-nav__link {
+    color: var(--color-white);
+  }
+
+  .main-nav {
+    background-color: var(--white);
+    align-items: center;
+    padding: 1rem;
+    margin: 0 auto;
+    gap: 1.7rem;
+    justify-content: space-around;
+  }
+
+  .main-nav .logo {
+    visibility: visible;
+    position: static;
+    cursor: pointer;
+  }
+
+  .main-nav__list {
+    display: flex;
+    flex-direction: row;
+    gap: 1.2rem;
+    position: static;
+    font-size: 1rem;
+    visibility: visible;
+    background-color: transparent;
+    height: max-content;
+    width: max-content;
+  }
+
+  #tecno {
+    color: var(--primary-color);
+    border: 4px solid var(--primary-color);
+    padding: 0.5em;
+    visibility: visible;
+    position: static;
+  }
+
+  .XIcon {
+    visibility: hidden;
+    position: absolute;
+  }
+
+  .main-nav__list__mobile {
+    visibility: hidden;
+    position: absolute;
+  }
+
+  .main-nav__list li a {
+    color: var(--gray-black);
+    font-family: "Lato", sans-serif;
+  }
+
+  main {
+    height: 100vh;
+    padding: 10vw;
+  }
```

Comment:
I kindly suggest you use flexbox to align elements in the center of the container